### ISSUE TITLE
Thanks for your work,excellent! some question about yolo-world finetune freeze, glip Pseudo label and prompt.

### ISSUE THREAD

**Issue by Meicsu199345** [AILab-CVC/YOLO-World#369]
### Question

Thanks for your work,excellent! 
**background**
I have a dataset about risk_control  100classes 200k images and 400k bboxes。
class such as
1:vulgar tongue kiss
2:porn bed photos
3:human smoking scenes

i want use it not only detect person ,car,  human smoking  in bed, so i need Re-parameterization?

**question**
Q1: can i get some suggestion about my method is correct?
  - change 1 update torch >=2.0.0 WARNING ⚠️ Upgrade to torch>=2.0.0 for deterministic training.
  - change 2  cache='disk'
  - change 3  load_model yolov8s-worldv2.pt(maybe need freeze)
  - change 4  freeze=10 (backbone),freeze only in backbone or all model is better for finetune?
  the paper show
![image](https://github.com/AILab-CVC/YOLO-World/assets/16251955/2700e334-6a7b-45b2-9d81-a6e05b6b660b)
  - add 10% background image

Q2:how can i use yolo world to do prompt?
![image](https://github.com/AILab-CVC/YOLO-World/assets/16251955/5fbefd8b-ce97-484e-8b9c-837dbdc6bc53)

Q3:how can i get Pseudo label（bboxes）by glip use this github-rep?
![image](https://github.com/AILab-CVC/YOLO-World/assets/16251955/f5e2626c-aea8-490c-a03e-c2005719fd0a)

Q4:how can i do A.1. Re-parameterization for RepVL-PAN use ultralytics?
![image](https://github.com/AILab-CVC/YOLO-World/assets/16251955/482423a5-f3be-43af-8efc-7627f658df4d)

thanks a lot, yolo world let my study easy~

**Comment by wondervictor** [comment]
Hi @Meicsu199345, sorry for the late reply!! Thanks for your interest in YOLO-World!
1. Using `torch~=1.13` and `requirements.txt` to install is fine and we'll remove the support for torch~2.0.
2. We only freeze the CLIP backbone for fine-tuning, other parameters are trainable. If you have a large vocabulary, you can also make CLIP trainable.
3. For prompt tuning, you can refer to [docs/prompt_yolo_world](https://github.com/AILab-CVC/YOLO-World/blob/master/docs/prompt_yolo_world.md). You just need to precompute the text embeddings and use the `SimpleYOLOWorldDetector` now.
4. The pseudo labels are listed here: [docs/data](https://github.com/AILab-CVC/YOLO-World/blob/master/docs/data.md#cc3m-pseudo-annotations).
5. The repo is independent from Ultralytics and we only support re-parameterization in this repo.

**Comment by Meicsu199345** [comment]
> Hi @Meicsu199345, sorry for the late reply!! Thanks for your interest in YOLO-World!
> 
> 1. Using `torch~=1.13` and `requirements.txt` to install is fine and we'll remove the support for torch~2.0.
> 2. We only freeze the CLIP backbone for fine-tuning, other parameters are trainable. If you have a large vocabulary, you can also make CLIP trainable.
> 3. For prompt tuning, you can refer to [docs/prompt_yolo_world](https://github.com/AILab-CVC/YOLO-World/blob/master/docs/prompt_yolo_world.md). You just need to precompute the text embeddings and use the `SimpleYOLOWorldDetector` now.
> 4. The pseudo labels are listed here: [docs/data](https://github.com/AILab-CVC/YOLO-World/blob/master/docs/data.md#cc3m-pseudo-annotations).
> 5. The repo is independent from Ultralytics and we only support re-parameterization in this repo.

Thank you very much for your answer～

**Comment by toobatehreem** [comment]
> Hi @Meicsu199345, sorry for the late reply!! Thanks for your interest in YOLO-World!
> 
> 1. Using `torch~=1.13` and `requirements.txt` to install is fine and we'll remove the support for torch~2.0.
> 2. We only freeze the CLIP backbone for fine-tuning, other parameters are trainable. If you have a large vocabulary, you can also make CLIP trainable.
> 3. For prompt tuning, you can refer to [docs/prompt_yolo_world](https://github.com/AILab-CVC/YOLO-World/blob/master/docs/prompt_yolo_world.md). You just need to precompute the text embeddings and use the `SimpleYOLOWorldDetector` now.
> 4. The pseudo labels are listed here: [docs/data](https://github.com/AILab-CVC/YOLO-World/blob/master/docs/data.md#cc3m-pseudo-annotations).
> 5. The repo is independent from Ultralytics and we only support re-parameterization in this repo.

How to fine-tune CLIP?

**Comment by Meicsu199345** [comment]
来信已达，定当拜读，谢谢了。。。
